experimental/components/ConversationalCore/language-generation/en-us/common.en-us.lg

```diff
@@ -0,0 +1,164 @@
+- IF: ${user.name && rand(0, 1000) > 500}
+    - ${concat(' ', user.name)}
+- ELSE:
+    - 
+# HowCanIHelp
+- What can I help you with?
+- What can I do for you?
+- How can I help you?
+- How can I help?
+
+
+# YouCanAskMeCard
+[Activity
+    Attachments = ${json(YouCanAskMeCardJson())}
+]
+
+# YouCanAskMeCardJson
+- ```
+{
+    "type": "AdaptiveCard",
+    "version": "1.0",
+    "body": [
+        {
+            "type": "Container",
+            "items": [
+                {
+                    "type": "TextBlock",
+                    "text": "You can ask me...",
+                    "size": "Large",
+                    "weight": "Bolder"
+                }
+            ]
+        },
+        {
+            "type": "Container",
+            "items": [
+                {
+                    "type": "ColumnSet",
+                    "columns": [
+                        {
+                            "type": "Column",
+                            "width": 10,
+                            "items": [
+                                {
+                                    "type": "Image",
+                                    "altText": "",
+                                    "url": "https://raw.githubusercontent.com/ryanlengel/adaptivecards/master/assets/icons/f4dd-32.svg",
+                                    "horizontalAlignment": "Left"
+                                }
+                            ]
+                        },
+                        {
+                            "type": "Column",
+                            "width": 90,
+                            "items": [
+                                {
+                                    "type": "TextBlock",
+                                    "text": "\"What's new?\"",
+                                    "size": "Medium",
+                                    "color": "Accent"
+                                }
+                            ],
+                            "selectAction": {
+                                "type": "Action.Submit",
+                                "title": "What's new?",
+                                "data": "What's new?"
+                            }
+                        }
+                    ],
+                    "separator": true
+                }
+            ],
+            "bleed": true,
+            "separator": true
+        },
+        {
+            "type": "Container",
+            "items": [
+                {
+                    "type": "ColumnSet",
+                    "columns": [
+                        {
+                            "type": "Column",
+                            "width": 10,
+                            "items": [
+                                {
+                                    "type": "Image",
+                                    "altText": "",
+                                    "url": "https://raw.githubusercontent.com/ryanlengel/adaptivecards/master/assets/icons/e787-32.svg",
```

Comment:
This needs to be cleaned up, likely when having an official design/spec per #435

Reply:
Will create a tracking issue.

Reply:
Created https://github.com/microsoft/botframework-components/issues/452.